### PR TITLE
Add tests for all of the simple Ada control-flow structures

### PR DESCRIPTION
**Pull request by martin-cs**
Not featured are exceptions and the restricted gotos that Ada has.

### CODE REVIEW

File: testsuite/gnat2goto/tests/cfg_test_11/cfg_test_11.adb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	     exit Inner when I - J - K = 17;
+	     
+	     Y := Y + I - J + K;
+	     
+	     exit Middle when I + J - K = 23;
+	     
+	     K := K + 1;
+	     
+	     exit Outer when I + J + K = 42;
+	  end loop Inner;
```
Comment (johnfxgalea) [inline testsuite/gnat2goto/tests/cfg_test_11/cfg_test_11.adb:29]: Is the indentation correct here?

File: testsuite/gnat2goto/tests/cfg_test_11/cfg_test_11.adb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	  
+	  exit Outer when J - X - I = 19;
+	  
+	  J := J + 1;
+       end loop Middle;
+       
+
```
Comment (johnfxgalea) [inline testsuite/gnat2goto/tests/cfg_test_11/cfg_test_11.adb:36]: Many empty lines.. Is this done on purpose?

File: testsuite/gnat2goto/tests/cfg_test_09/cfg_test_09.adb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+procedure CFG_Test_09 (X : in Integer; I : in out Integer) is
+begin
+   I := 0;
+   
+   if X > 0 then
+      loop
+	 exit when I - X > 20;
```
Comment (johnfxgalea) [inline testsuite/gnat2goto/tests/cfg_test_09/cfg_test_09.adb:7]: Is Indentation correct here?

**Review by johnfxgalea — APPROVED** [review]

File: testsuite/gnat2goto/tests/cfg_test_09/cfg_test_09.adb (hunk 3 above)
Comment (martin-cs) [reply testsuite/gnat2goto/tests/cfg_test_09/cfg_test_09.adb:7]: It is not.  Will fix.

File: testsuite/gnat2goto/tests/cfg_test_11/cfg_test_11.adb (hunk 1 above)
Comment (martin-cs) [reply testsuite/gnat2goto/tests/cfg_test_11/cfg_test_11.adb]: I have used `gnatpp` to pretty print the code.  It looks better to me now.

File: testsuite/gnat2goto/tests/cfg_test_11/cfg_test_11.adb (hunk 2 above)
Comment (martin-cs) [reply testsuite/gnat2goto/tests/cfg_test_11/cfg_test_11.adb]: I have used `gnatpp` to pretty print the code.  It looks better to me now.

File: testsuite/gnat2goto/tests/cfg_test_09/cfg_test_09.adb (hunk 3 above)
Comment (martin-cs) [reply testsuite/gnat2goto/tests/cfg_test_09/cfg_test_09.adb]: I have used `gnatpp` to pretty print the code.  It looks better to me now.

**Pull request merged**